.coveragerc

```diff
@@ -1,6 +1,16 @@
 [report]
+# Regexes for lines to exclude from consideration
+exclude_lines =
+    # Don't complain about missing debug-only code:
+    def __repr__
```

Comment:
This could be useful to test.

Reply:
@fchollet, Updated.

Reply:
I would like to remind you that `.coveragerc` is about only calculating test coverages. The `exclude_lines` is not excluding related tests but excluding related missing line counts for reporting test coverages.